src/libtorchaudio/lfilter.cpp

```diff
@@ -100,194 +100,19 @@ void lfilter_core_generic_loop(
   }
 }
 
```

Comment:
We've got `lfilter_core_generic_loop` left-over above in this file. Is it still needed?